repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/guidance-content.ftl

```diff
@@ -96,6 +96,9 @@
                             </#list>
                         </div>
                     </#if>
+                    <div class="nhsuk-grid-column-full nhsuk-section__content">
```

Comment:
Could we wrap this up with `<#if guidanceDocument.relatedContent??>...</#if>` as `guidanceDocument.relatedContent` is an optional field ?

Reply:
done.